post-scan-actions/gcp-python-slack-notification/README.md

```diff
@@ -0,0 +1,99 @@
+# Cloud One File Storage Security Post Scan Action for GCP - Slack notifications
+
+
+## Prerequisites
+
+1. **Install supporting tools**
```

Comment:
If you are installing the Google Cloud SDK, why not say it as opposed to supporting tools? And why is this bold?
```suggestion
1. Install Google Cloud SDK.
```